### PR TITLE
ZUR-135 My PR adds a feature "removed billings and pricing from contact Page"

### PR DESCRIPTION
**Pull request by arcteggzz**
slack username: arcteggzz
Linear Ticket ID: ZUR-135
Linear Ticket Link: https://linear.app/zurichat2/issue/ZUR-135/remove-the-pricing-section-in-the-contact-page
Linear Ticket instruction: "Remove the pricing section on the contact page"

I removed the billings and pricing section from the Contact Page.

PR-ContactPageScreenshot

### CODE REVIEW

File: .gitpod.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# This configuration file was automatically generated by Gitpod.
+# Please adjust to your needs (see https://www.gitpod.io/docs/config-gitpod-file)
+# and commit this file to your remote git repository to share the goodness with others.
+
+tasks:
+  - init: yarn install && yarn run build
+    command: yarn run start
+
+
```
Comment (JulianaSau) [inline .gitpod.yml:9]: Please remove this `gitpod.yml` configuration file from your commits

**Review by JulianaSau — CHANGES_REQUESTED** [review]

**Comment by arcteggzz** [comment]
@Sau 
I have removed the file from the commit.

**Review by JulianaSau — APPROVED** [review]

**Comment by JulianaSau** [comment]
Pull updates from the repo so that I can merge

**Comment by arcteggzz** [comment]
I have pulled updates from (https://github.com/zurichat/zc_main) [main branch]. 
It is up to date right now.

So you can merge now.

**Comment by JulianaSau** [comment]
You're supposed to pull changes from the dev branch and push them to your remote repository so that it reflects in this pr

**Comment by arcteggzz** [comment]
I have pulled updates from (https://github.com/zurichat/zc_main) [dev branch].
It is up to date right now.

So you can merge now.

**Pull request merged**